File: OOP_Tasks/OOP_Task1/JAVA/Main.java
package org.example;

import java.util.Scanner;

public class Main
{
    public static void main(String[] args)
    {
        Scanner scanner = new Scanner(System.in);
        char o;
        int x, y;

        while (true)
        {
            System.out.print("Enter first number: ");
            x = scanner.nextInt();

            System.out.print("Enter operation: ");
            o = scanner.next().charAt(0);

            System.out.print("Enter second number: ");
            y = scanner.nextInt();

            switch (o)
            {
                case '+':
                    System.out.println("sum is " + myMath.Sum(x, y));
                    break;
                case '-':
                    System.out.println("sub is " + myMath.Sub(x, y));
                    break;
                case '*':
                    System.out.println("mul is " + myMath.Mul(x, y));
                    break;
                case '/':
                    System.out.println("div is " + myMath.Div(x, y));
                    break;
                case '%':
                    System.out.println("rem is " + myMath.Rem(x, y));
                    break;
                default:
                    System.out.println("Inavalid Operation");
                    break;
            }
            System.out.print("Another Operation? ");
            o = scanner.next().charAt(0);

            if (o != 'T')
                break;
        }
    }
}

File: OOP_Tasks/OOP_Task1/C#/main.cs
using System;
class MainProgram
{
  static void Main()
  {
      char o;
      int x, y;
      
      while (true)
      {
          Console.Write("Enter first number: ");
          x = int.Parse(Console.ReadLine());
          
          Console.Write("Enter operation: ");
          o = Console.ReadLine()[0];
          
          Console.Write("Enter second number: ");
          y = int.Parse(Console.ReadLine());
          
          switch (o)
          {
              case '+':
                Console.WriteLine("sum is " + MyMath.Sum(x, y));
                break;
              case '-':
                Console.WriteLine("sub is " + MyMath.Sub(x, y));
                break;
              case '*':
                Console.WriteLine("mul is " + MyMath.Mul(x, y));
                break;
              case '/':
                Console.WriteLine("div is " + MyMath.Div(x, y));
                break;
              case '%':
                Console.WriteLine("rem is " + MyMath.Rem(x, y));
                break;
              default:
                Console.WriteLine("Inavalid Operation");
                break;
          }
          Console.Write("Another Operation? ");
          o = Console.ReadLine()[0];
          
          if (o != 'T')
            break;
      }
      
      
  }
}
